static/css/restyle.less

```diff
@@ -904,21 +918,35 @@ button.good {
     text-transform: uppercase;
   }
 
-  li a:link {
-    color: @link-on-white-bg !important;
-    font-size: 12px !important;
+  li a {
+    color: @link-on-white-bg;
+    font-size: 12px;
     font-weight: normal;
-    padding: 6px;
+    padding: 5px 6px;
+    position: relative;
+    text-decoration: none;
 
     &:after {
+      color: #dadada;
+      color: rgba(0, 0, 0, 0.125);
       content: '\203a';
-      float: right;
-      font-size: 10px;
+      font-family: Georgia;
```

Comment:
Will less convert it? I actually prefer doing fonts like this; I find the shorthand a bit confusing.